client/src/components/Overview/Overview.jsx

```diff
@@ -14,7 +14,8 @@ import ImageGallery from './ImageGallery';
 import StyleSelector from './StyleSelector';
 
 function Overview() {
-  const [productId, setProductId] = useState(Math.floor(Math.random() * (38321 - 37311) + 37311));
+  // const [productId, setProductId] = useState(Math.floor(Math.random() * (38321 - 37311) + 37311));
+  const [productId, setProductId] = useState(38108)
```

Comment:
This can stay for now, as we're testing and tweaking, but eventually it will have to be pulled from the Context data store